.circleci/config.yml

```diff
@@ -1,21 +1,73 @@
-version: 2
+version: 2.1
+
+orbs:
+  node: circleci/node@4.4.0
+
+workflows:
+  version: 2
+  workflow:
+    jobs:
+      - build:
+          matrix:
+            parameters:
+              # We should test against "latest Node 16", but there's an issue with 16.2
+              # This will be updated soon, along with a bug report to Node.js
+              # node-version: ["lts/erbium", "lts/fermium", "16"]
+              node-version: ["lts/erbium", "lts/fermium", "16.1"]
```

Comment:
This felt reasonable - currently supported "latest" LTS versions and "current" (see comment above about why it's not just "latest 16".